### DISCUSSION TITLE
Поддержка raid

### DISCUSSION THREAD

**Discussion by mesb1** [mikopbx/Core#63] [Ideas (Идеи)]
Для случая установки станции на обычные железки с парой дисков, хотелось бы из коробки возможность сразу установиться в raid1 для какой-никакой отказоустойчивости.
Речь про софт рейд с помощью mdadm.

**Comment by boffart** [comment]
Этого нет в планах. Не вижу смысла. Используйте  виртуализацию, если необходима отказоустойчивость.
Пример - esxi.

**Reply by mesb1** [reply]
> Этого нет в планах. Не вижу смысла. Используйте виртуализацию, если необходима отказоустойчивость.
> Пример - esxi.

Какого-то кластерного режима тоже не планируется?
Будь то вмки или реальное железо.

**Reply by boffart** [reply]
> Какого-то кластерного режима тоже не планируется?
> Будь то вмки или реальное железо.

Что Вы под этим понимаете?
 
Сама суть **MIKOPBX** - простота настройки и поддержки. Все сложное и непонятное выпиливаем и добавляем в крайнем случае. 
Если начнем городить все подряд : openvpn / raid / chan_dongle / dahdi платы и т.п. и т.д.  - получится elastix. 

Если для вас это все критично и необходимо - пишите на help@miko.ru - рассмотрим вариант разработки платного доп. модуля.

**Reply by mesb1** [reply]
Пропустил уведомление.
У меня в голове некоторый аналог того как это действует в pfsense.
То есть станции синкаются между собой конфигом, при умирании ведущей, в актив переходит ждущая.

**Comment by boffart** [comment]
В новой версии АТС добавим поддержку "[Установка на MDADM RAID1](https://wiki.mikopbx.com/faq:mdraid1)" 
Если интересно, могу выслать ссылку на тестовую сборку.

**Reply by mesb1** [reply]
Да, давайте попробуем.

**Reply by boffart** [reply]
https://files.miko.ru/s/QAotQGbKxTPGiCf

**Reply by mesb1** [reply]
> В новой версии АТС добавим поддержку "[Установка на MDADM RAID1](https://wiki.mikopbx.com/faq:mdraid1)"


Первый момент:
В этой инструкции нужно указать необходимо ли дождаться синхронизации, статус которой виден:
cat /proc/mdstat

Ладно у меня ссд, но подобный сетап кмк чаще будет оказываться на жёстких дисках.

**Comment by mesb1** [comment]
После установки как в консоли так и после логона по ссх показывается надпись красным:

System integrity is broken   

Ещё я следуя упоминанию на страничке про груб - хотел посмотреть конфиг или сделать груб-апдейт для второго диска, но чото както ненашёл.
